app/navigation/TrusteeOrganisationNavigator.scala

```diff
@@ -14,22 +14,43 @@
  * limitations under the License.
  */
 
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package navigation
 
+import java.lang.ModuleLayer.Controller
+
 import config.FrontendAppConfig
 import controllers.register.trustees.organisation.routes._
+import controllers.register.trustees.organisation.mld5.{routes => mld5}
 import models.ReadableUserAnswers
 import pages.Page
 import pages.register.trustees.organisation._
+import pages.register.trustees.organisation.mld5._
 import play.api.mvc.Call
 
 class TrusteeOrganisationNavigator extends Navigator {
 
   override def simpleNavigation(draftId: String): PartialFunction[Page, ReadableUserAnswers => Call] = {
     case NamePage(index) => _ => UtrYesNoController.onPageLoad(index, draftId)
-    case UtrPage(index) => _ => CheckDetailsController.onPageLoad(index, draftId)
+    case UtrPage(index) => ua => mld5NavUTRYesNo(draftId, index, ua.is5mldEnabled)
```

Comment:
I'm confused as to why this line points to a different method than line 62. The UTR page and answering no on the UTR yes/no page should go to the same place. Is there something wrong in the unit tests?